twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx

```diff
@@ -168,8 +169,8 @@ export default (props: Props) => {
       )}
       <div style={{ flex: 1 }} />
       <div
-        className={'option ' + (props.inputValue ? '' : 'disabled ')}
-        onClick={() => props.onSend && props.onSend()}
+        className={'option ' + (props.inputValue || !props.isEmpty ? '' : 'disabled ')}
+        onClick={() => !props.isEmpty && props.onSend && props.onSend()}
```

Comment:
Remove the props.inputValue because it is used nowhere else.
Only use the props.isEmpty

Reply:
Removed :+1: